### PR TITLE
My assignment 4

### PR DESCRIPTION
**Pull request by koushikalakhamraju**
Screenshot (84)

### CODE REVIEW

File: src/Assignment-4/assignment4.css [hunk 1]
```diff
@@ -5,3 +5,117 @@
...
+    border: 5px solid #3F8DA7;
+    text-decoration: none;
+    padding: 5px 10px;
+    color: #3F8DA7;
+}
+#hero a:hover{
+    background: #3F8DA7;
+    color: white;
+}
+.three-col {
```
Comment (praveendhakade) [inline src/Assignment-4/assignment4.css:69]: You can try to center the three-col to align with header and footer

Comment (koushikalakhamraju) [reply src/Assignment-4/assignment4.css:69]: ok. i will make the above modifications

**Comment by koushikalakhamraju** [comment]
Screenshot (86)

**Review by praveendhakade — APPROVED** [review]
Great!

**Comment by SuneetKumar19** [comment]
Good work!

**Pull request merged**